simulation_ws/src/cloudwatch_simulation/launch/world_turtlebot_navigation.launch.py

```diff
@@ -0,0 +1,99 @@
+# *******************************************************************************/
```

Comment:
I think this file should be called `sample_world_turtlebot_navigation.launch.py`. `world_turtlebot_navigation.launch.py` is still too generic, since our sample worlds are guaranteed to still have occupancy maps and stuff.

Reply:
sounds good